generate.go

```diff
@@ -53,7 +53,14 @@ func (zp *ZoneParser) generate(l lex) (RR, bool) {
 		return zp.setParseError("bad range in $GENERATE range", l)
 	}
 
-	zp.c.Next() // _BLANK
+	// _BLANK
+	if l, ok := zp.c.Next(); ok {
+		if l.value != zBlank {
+			return zp.setParseError("Expect blank after $GENERATE range", l)
```

Comment:
to be more in sync with the rest of the parsing errors, this should be something ala "garbage after $GENERATE range" (same for the other error), might also drop the if-else in that case (most errors aren't super descriptive)